src/panels/lovelace/editor/structs/entities-struct.ts

```diff
@@ -14,6 +14,8 @@ export const entitiesConfigStruct = union([
     tap_action: optional(actionConfigStruct),
     hold_action: optional(actionConfigStruct),
     double_tap_action: optional(actionConfigStruct),
+    show_icon: optional(boolean()),
+    show_name: optional(boolean()),
```

Comment:
do all entity configs support this?

Reply:
I took that over from Zack here (https://github.com/home-assistant/frontend/pull/12792), but I don't think all support it yet. In the documentation we mention it for button cards, button rows in entities card, glance card and badges.

So I changed it to a more specific new one derived from `buttonsEntitiesRowConfigStruct`.